frontend/src/app/redux/course/course.sagas.jsx

```diff
@@ -16,3 +16,17 @@ export function* createCourse({ name, description }) {
         yield put(CourseActions.createCourseFailure());
     }
 }
+
+export function* updateCourse({ id, name, description }) {
+    try {
+        yield put(CourseActions.loading(true));
+        const response = yield call(axios.put, `/courses/${id}`, { name, description });
+        if (response.status === 200) {
+            yield put(CourseActions.updateCourseSuccess());
```

Comment:
In `course.redux.jsx` you can see that the success action takes a `data` parameter.
Why are we not passing one here?

It seems unclear to me whether that `data` thingy is needed in the first place. And if it exists, why are we passing `undefined` to it?

Reply:
Yeah, sorry about that. The data is intended to modify the state, but in case of these actions the state doesn't suffer any modifications. I'll remove it as it is useless right now.